app/views/govuk_publishing_components/components/_layout_footer.html.erb

```diff
@@ -18,9 +18,15 @@
                 <% item[:items].each do |item| %>
                   <% if item[:href] && item[:text] %>
                     <li class="govuk-footer__list-item">
-                      <a class="govuk-footer__link" href="<%= item[:href] %>">
-                        <%= item[:text] %>
-                      </a>
+                      <% if item[:target].nil? %>
+                        <%= link_to item[:text], item[:href], target: '',
+                          class: "govuk-footer__link"
+                        %>
+                      <% else %>
+                        <%= link_to item[:text], item[:href], target: item[:target],
```

Comment:
There are some security considerations to make with opening links in new windows: https://developers.google.com/web/tools/lighthouse/audits/noopener

Additionally we often avoid opening links in new windows because of accessibility issues associated with doing so, but I'm not sure what the user need is.

Reply:
I agree with @nickcolley, if we're going to allow target `_blank`s, we should at least add `rel="noopener"`.